python/foglamp/tasks/north/sending_process.py

```diff
@@ -819,6 +855,7 @@ def _retrieve_configuration(self, cat_name=None, cat_desc=None, cat_config=None,
             if is_stream_valid is False:
                 raise ValueError("Error in Stream Id for Sending Process {}".format(self._name))
             self.statistics_key = await self._get_statistics_key()
+            self.master_statistics_key = await self._get_master_statistics_key()
```

Comment:
There is no need for _get_master_statistics_key() method. All new keys are fixed keys like "READINGS" etc, hence should come in init.sql.

Reply:
No, we are moving away from this mechanism, hence these new keys are not done this way.

Reply:
Then in that case a bypass mechanism will be needed for missing keys in ping method, otherwise ping will continue to fail and will affect start of the foglamp server by "foglamp start" call also. Also health check call will continue to fail.